presidio-analyzer/presidio_analyzer/analyzer_engine.py

```diff
@@ -242,6 +245,28 @@ def __remove_low_scores(
         new_results = [result for result in results if result.score >= score_threshold]
         return new_results
 
+    def __ensure_recognition_metadata(
```

Comment:
nit: one underscore (saw you did the same in the other class)

Reply:
nit: missing a bit descriptiveness IMO, ensure what?
maybe ensure_name_in_recognition_metadata or ensure_recognizer_name_in_metadata or add_recgnizer_name_if_not_exists

Reply:
> nit: missing a bit descriptiveness IMO, ensure what? maybe ensure_name_in_recognition_metadata or ensure_recognizer_name_in_metadata or add_recgnizer_name_if_not_exists

I've changed it to "__add_recognizer_name_if_not_exitsts"